dist/date-picker/date-picker.css

```diff
@@ -0,0 +1,114 @@
+.date-picker {
+  background-color: var(--date-picker-background-color, var(--color-background-primary));
+  border-color: var(--date-picker-border-color, var(--color-stroke-default));
+  border: 1px solid;
+  border-radius: 16px;
+  box-shadow: 0 0 16px 0 rgba(0, 0, 0, 0.35);
+  display: flex;
+  gap: 16px;
+  padding: 16px;
+  position: absolute;
+}
+.date-picker[hidden] {
+  display: none;
+}
+.date-picker__month h3 {
+  line-height: 20px;
+  margin: 0 0 16px;
+  text-align: center;
+}
+.date-picker--single-month .date-picker__month h3 {
+  text-align: left;
+}
+.date-picker__month-button.icon-btn {
+  background-color: transparent;
+  height: 32px;
+  min-width: 32px;
+  position: absolute;
```

Comment:
** Please Continue Conversation ** 

I had the buttons in the header originally with different styles, but opted to position absolutely instead because of the differences between single and double pane date picker. I think consistent HTML is more important than floating styles, but I may be wrong. Please give me your thoughts @agliga

Reply:
Imo consistent html is the right way to go
But consistent in terms of structure
For example the way it should be is

Go to prev month - header
Or
Header - go to next month
Or (single month view)
Header - go to prev - go to next

The elements should be in the right order
We shouldn't move the buttons outside of where they should be and style them. 

So the structure should change based on what the user wants. If the button is before the text, it should be before the text in the markup. If after then after etc

Reply:
Yes! We should opt to allow HTML dictate structure and the order should be preserved, otherwise natural tabbing order will get compromised and become an a11y concern.